drink/test-macro/src/ir.rs

```diff
@@ -0,0 +1,82 @@
+use darling::{ast::NestedMeta, FromMeta};
+use proc_macro2::TokenStream as TokenStream2;
+use syn::ItemFn;
+
+/// The macro arguments (available configuration).
```

Comment:
Arguments or attributes? :) Maybe I'm missing the nomenclature but we're passing the `attr` arg to `MacroArgs::from_list` - is it OK to use either `Args` or `Attr` everywhere?

Reply:
kinda... I will unify it though